README.md

```diff
@@ -31,13 +31,13 @@ header:
 
   comment: on-failure
 
-# If you want to check dependencies' license compatibility, uncomment the following section
-# dependency:
-#   files:
-#     - pom.xml           # If this is a maven project.
-#     - Cargo.toml        # If this is a rust project.
-#     - package.json      # If this is a npm project.
-#     - go.mod            # If this is a Go project.
+# If you don't want to check dependencies' license compatibility, remove the following part
+dependency:
```

Comment:
Previously when users uncomment the the codes they might make the indentation wrong, so I uncomment these by default and users can opt-out by deleting them